src/TRestTrackAlphaAnalysisProcess.cxx

```diff
@@ -0,0 +1,281 @@
+///______________________________________________________________________________
+///______________________________________________________________________________
+///
+///
+///             RESTSoft : Software for Rare Event Searches with TPCs
+///
+///             TRestTrackAlphaAnalysisProcess.cxx
+///
+///_______________________________________________________________________________
```

Comment:
Please, add the official headers. See for example `TRestGeant4Metadata` style. Add also some documentation, include figures if possible.